StrongReferenceDemo/NodeCell.swift

```diff
@@ -10,9 +10,7 @@ import UIKit
 
 class NodeCell: UITableViewCell {
     var node: Node!
-
```

Comment:
Since the original purpose of this property was to demonstrate the Pattern 2 leak, and you've moved that pattern to a much better implementation, let's take this text out altogether and just have the cell display the name rather than "status text".